_data/engine-cli/docker_container_exec.yaml

```diff
@@ -1,7 +1,7 @@
 command: docker container exec
 short: Run a command in a running container
 long: Run a command in a running container
-usage: docker container exec [OPTIONS] CONTAINER COMMAND [ARG...]
+usage: docker container exec [OPTIONS] CONTAINER COMMAND [ARG...] [flags]
```

Comment:
Opened a PR with a fix; https://github.com/docker/cli/pull/1539

Reply:
Let me try getting them merged; if you can then regenerate from the 18.09 release _branch_ (instead of the tag), then it should be fixed. DM'ing some people on Slack now 😅

Reply:
@seemethere https://github.com/docker/cli/pull/1540 was merged, so if you can update using the 18.09 release branch, we should be good to go 🤗